src/components/PageCrud/PageCrudData.js

```diff
@@ -1,105 +1,58 @@
-import { useQuery, useQueryClient, useMutation } from '@tanstack/react-query'
-import { useNavigate, useLocation } from 'react-router-dom'
+import { useNavigate, useLocation, useSearchParams } from 'react-router-dom'
 
 // FPCC
-import { DOC_PAGE } from 'common/constants'
-import api from 'services/api'
 import { useSiteStore } from 'context/SiteContext'
-import pageCrudDataAdaptor from 'components/PageCrud/pageCrudDataAdaptor'
-import { useNotification } from 'context/NotificationContext'
-import { selectOneMediaFormHelper } from 'common/utils/mediaHelpers'
+import {
+  usePage,
+  usePageCreate,
+  usePageInfoUpdate,
+  usePageDelete,
+} from 'common/dataHooks/usePages'
+import { IMAGE, VIDEO } from 'common/constants'
 
 function PageCrudData() {
   const { site } = useSiteStore()
   const navigate = useNavigate()
   const location = useLocation()
-  const queryClient = useQueryClient()
-
-  const { setNotification } = useNotification()
-
   const backHandler = () => navigate(-1)
 
-  const _pageId = new URLSearchParams(location.search).get('id')
-    ? new URLSearchParams(location.search).get('id')
-    : null
+  const [searchParams] = useSearchParams()
+  const pageId = searchParams.get('id') || null
```

Comment:
If we're switching over to fetching by slug and driving the component using the `slug` search param then we will need to update the edit link found on line 42 of `PageCrudPresentation` . We can then remove this retrieval of the id from searchParams.